src/shapes/Group.ts

```diff
@@ -28,6 +28,18 @@ import {
   LAYOUT_TYPE_REMOVED,
 } from '../LayoutManager/constants';
 
+/**
+ * This class is created to handle the specific case of the group fromObject.
+ * Is not meant to be used anywhere outside the fromObject function.
+ * We could have used a boolean in the constructor but we think the boolean
+ * would stay in the group's constructor interface and create confusion.
+ * The point of this LayoutManager is really just to don't touch anything.
+ */
+class RespectfulLayoutManager extends LayoutManager {
```

Comment:
Can we rename this?
DehydrationLayoutManager/NoopLayoutManager/FromObjectLayoutManager?

Reply:
is an internal name is not moving from this file, it doesn't really matter. i can call it Noop yes

Reply:
renamed to NoopLayoutManager